generated/prisma-client/prisma-schema.js

```diff
@@ -614,6 +640,11 @@ input ClassroomUpdateWithWhereUniqueWithoutSchoolInput {
   data: ClassroomUpdateWithoutSchoolDataInput!
 }
 
+input ClassroomUpsertNestedInput {
```

Comment:
You have a typo

Reply:
This is a generated file by Prisma. It is correct